### PR TITLE
Fixed tests, refactor from prototype to class, updated test npm script

### PR DESCRIPTION
**Pull request by JoeKarlsson**
- The test were not initializing correctly, so I nested them in a "beforeEach" hook to instantiate a new matrix before each test. 
- I also converted the prototypes into classes.
- I updated the Dev Dependencies (there were none).
-  I made an npm test script :dancer:

### CODE REVIEW

File: MatrixRotator.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
+const Direction = require("./Direction").Direction;
```
Comment (theRemix) [inline MatrixRotator.js:1]:
```javascript
const { Direction } = require('./Direction');
```

File: test/MatrixRotator-spec.js [hunk 2]
```diff
@@ -1,35 +1,72 @@
-var chai = require('chai');
+const chai = require('chai');
 chai.should();
-var MatrixRotator = require("../MatrixRotator").MatrixRotator;
-var Matrix = require("../Matrix");
-var Direction = require("../Direction").Direction;
+const MatrixRotator = require("../MatrixRotator");
+const Matrix = require("../Matrix");
+const Direction = require("../Direction").Direction;
```
Comment (theRemix) [inline test/MatrixRotator-spec.js]:
```javascript
const { Direction } = require('./Direction');
```

**Review by theRemix — CHANGES_REQUESTED** [review]
suggesting alternative const declaration

**Comment by JoeKarlsson** [comment]
Fixed the object destructing. :taco:

**Pull request merged**